### PR TITLE
add to_h method to take unformatted list of env vars and format them

### PR DESCRIPTION
**Pull request by chrunn**

### CODE REVIEW

File: lib/config_hub/client.rb [hunk 1]
```diff
@@ -37,6 +37,14 @@ def fetch(key)
       end
     end
 
+    def to_h
+      pull
```
Comment (chrunn) [inline lib/config_hub/client.rb]: @fractallian , question about the use of `pull` here, is this appropriate?

File: spec/config_hub/client_spec.rb [hunk 2]
```diff
@@ -62,6 +62,14 @@
       end
     end
 
+    describe '#to_h' do
+      it 'returns a hash of the keys to values from the properties' do
+        expect(subject.to_h).to include(
+          'salesforce.web_host' => 'http://scheduling.example.com',
+          'thiskeyisnil' => nil
+        )
```
Comment (josephgrossberg) [inline spec/config_hub/client_spec.rb]: Nit: Can we be more precise than `include`? Are the `Hash` contents unpredictable?

File: lib/config_hub/client.rb (hunk 1 above)
Comment (fractallian) [reply lib/config_hub/client.rb]: no, if config has not been pulled, we should raise a ConfigNotPulledError

**Review by fractallian — APPROVED** [review]
peer reviewed

File: spec/config_hub/client_spec.rb [hunk 3]
```diff
@@ -62,6 +62,24 @@
...
+      describe 'when config has been pulled' do
+        before do
+          subject.pull
+        end
+
+        it 'returns a hash of the keys to values from the properties' do
+          expect(subject.to_h).to eq({
+            'salesforce.web_host' => 'http://scheduling.example.com',
+            'thiskeyisnil' => nil
+          })
```
Comment (josephgrossberg) [inline spec/config_hub/client_spec.rb:79]: 👍

**Pull request merged**